style/CONTRIBUTE.md

```diff
@@ -180,11 +180,15 @@ Highway shields are a key feature of the OpenStreetMap Americana style. This gui
 
 In addition, the following style guidelines are recommended:
 
-1. Use a 1px stroke size for line elements. Horizontal/vertical lines should be aligned to the pixel grid. In Inkscape, use Shift+Ctrl+D, and create a rectangular grid in the "Grids" tab and enable the "snap to grid" option.
+1. Use a 1px stroke size for line elements. Horizontal/vertical lines should be aligned to the pixel grid.
+   - In Inkscape, press `Ctrl+Shift+D`, and create a rectangular grid in the "Grids" tab. Set Spacing X and Spacing Y to `0.5`, and Major grid line every `2`. You can turn snapping to the grid on and off by pressing `%`.
 2. Complex or intricate graphic elements should be simplified.
 3. Background shading should be removed.
 4. Route numbers should be large enough to be legible.
-5. Shields should consistently use squared or rounded corners, but not a mix of both.
+5. Shields should consistently use squared or rounded corners, but not a mix of both. Rounded corners should have a 2px radius.
+   - To give a rectangle a 2px corner radius, drag the circular handles 2 pixels (4 grid lines) away from the edge, as shown in the top right:
+
+<img src="https://user-images.githubusercontent.com/1732117/167747120-308e9ea5-2f21-49e9-bb09-3b3309b519f9.png"/>
```

Comment:
Should be able to add files to doc-img and reference them with relative paths in markdown.